apps/verification-portal/src/lib/features/actions.ts

```diff
@@ -5,20 +5,7 @@ export const formSearch = {
   formSearch: async (event: ServerLoadEvent) => {
     const form = await event.request.formData();
     const collection = form.get('collection') as string;
-
-    let prefix = '';
-    switch (collection) {
-      case 'sub0':
-        prefix = 'did:asset:deep:polkadot.asset-hub:13:';
-        break;
-      case 'soundwaves':
-        prefix = `did:asset:eip155:42161.erc721:0x6cc7c9b2aa5fdcc044f9a51d9d083fd16aeb0a78:`;
-        break;
-      //TODO: Where is configuration for HH?
-      default:
-        throw new Error('Invalid collection id');
-    }
-
+    const prefix = `${collection}:`;
     const search = form.get('search') as string;
     const regex = new RegExp(`^${prefix}`);
     const isMatch = regex.test(search);
```

Comment:
Yes we can use DID for search, that was original design and so function presists - thought we changed the placeholder in the course of updates